pkgs/applications/networking/giara/default.nix

```diff
@@ -6,17 +6,19 @@
 , ninja
 , python3
 , wrapGAppsHook
```

Comment:
Can you submit an upstream merge request to use `gtk4-update-icon-cache` in https://gitlab.gnome.org/World/giara/-/blob/8d4850bb545f8ad9e40a9c3d235356db8794c8b6/meson_post_install.py#L12, then use `wrapGAppsHook4` instead?

Reply:
I don't have enough knowledge on this matter to feel comfortable submitting a patch upstream. What I did is to use substituteInPlace and change `gtk-update-icon-cache` to `gtk4-update-icon-cache` then use the `wrapGAppsHook4` as you suggested. The app still builds afterwards